### PR TITLE
fitch qoute and add loading and use the error stat

### PR DESCRIPTION
**Pull request by majdalkilany**
fitch quote
 add loading
 use the error state

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,46 @@
...
+        </h6>
+      );
+    }
+    return <h6>Loading...</h6>;
+  };
   return (
     <div>
       <Calculator />
+      {renderQuote()}
     </div>
```
Comment (abdulhamiid) [inline src/App.js]: Please kindly modify your code so as to match the project requirements 🤓

image

Kindly ensure that the new component is imported into the main component this way 👍 

E.g
```jsx
function App() {
  return (
    <>
      {/* Rest of JSX */}
      <Calculator />
      <Quotes />
    </>
  )
}
```

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @majdalkilany 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>

## Highlights 🌟

- ✔️ Use the correct React hook when fetching data
- ✔️ Connect to an API within a functional component
- ✔️ Show different UI outputs depending on the component state

## Required Changes ♻️
_Please kindly check the comments under the review._

[REQUIRED] - When I opened the live demo link in your README file, I got a blank page.

For a professional repository, I would suggest we remove the link since you do not have a deployed version of your project yet. 🤗

image


## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

### **OPTIONAL** 

The quote
image



- [ ] I noticed that your React component is rendering twice, and that is because you are running your app in strict mode. 

- [ ] I would suggest that you remove -> React.StrictMode, from ReactDOM.render code.

- [ ] Kindly go to index.js and removethe strict mode tag. You will find a single render.

This will render 2 times on every re-render: 👇
```jsx
ReactDOM.render(
  <React.StrictMode>
      <App />
  </React.StrictMode>,
  document.getElementById('root')
);
```

This will render 1 time 👇

```jsx
ReactDOM.render(
  <>
    <App />
  </>,
  document.getElementById('root')
);
```

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (majdalkilany) [reply src/App.js]: Hi, @abdulhamiid are these changes required or optional?

**Comment by majdalkilany** [comment]
fitch the data in the quote component and build the react app

**Review by codecaiine — APPROVED** [review]
Hi @majdalkilany , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Quotes are fetching as expected ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**